### PR TITLE
Clear A8

### PR DESCRIPTION
**Pull request by alindobx**
This PR clears the field A8.

### CODE REVIEW

**Comment by mentor-tara** [comment]
Looks like something is wrong :astonished:	Please see the CI checks below for more information. Click the "Details" link for more information as shown in this screenshot.

Screenshot of CI checks[](error)

**Comment by alindobx** [comment]
Tara can you please approve my request?

**Comment by mentor-tara** [comment]
Looks like something is wrong :astonished:	Please see the CI checks below for more information. Click the "Details" link for more information as shown in this screenshot.

Screenshot of CI checks[](error)

**Comment by alindobx** [comment]
When will you approve my changes?

**Comment by mentor-tara** [comment]
Looks like something is wrong :astonished:	Please see the CI checks below for more information. Click the "Details" link for more information as shown in this screenshot.

Screenshot of CI checks[](error)

File: commands.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
+
 {
-  "commands": []
+  "commands": [
+    "clear AB"
```
Comment (jkettmann) [inline commands.json]: I think you wrote AB but it should be A8 😄

**Review by alindobx — COMMENTED** [review]
Changes made

**Review by mentor-tara — APPROVED** [review]
### You're good. No mines hit. Merge this PR to continue the game.

This is the current status:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   |   | 2 | 2 | 3 | 2 |
| **3** |   |   |   | 2 | 1 | 0 | 0 | 0 |
| **4** | 1 | 2 | 1 | 1 | 0 | 0 | 0 | 0 |
| **5** | 0 | 0 | 0 | 0 | 0 | 0 | 1 | 1 |
| **6** | 0 | 0 | 0 | 1 | 1 | 1 | 2 |   |
| **7** | 0 | 1 | 1 | 2 |   |   |   |   |
| **8** | 0 | 1 |   |   |   |   |   |   |

Mines remaining: 10
Moves made: 1[](clear-A8)

**Pull request merged**